File: benchmarks/cross/progs/sieve.js
const N=2000000;
const flags=new Array(N+1).fill(true);
let count=0,i=2;
while(i<=N){
  if(flags[i]){ count++; let j=i*i; while(j<=N){ flags[j]=false; j+=i; } }
  i++;
}
console.log(count);

File: benchmarks/cross/progs/sieve.py
N = 2000000
flags = [True]*(N+1)
count = 0
i = 2
while i <= N:
    if flags[i]:
        count += 1
        j = i*i
        while j <= N:
            flags[j] = False
            j += i
    i += 1
print(count)
